requirements/requirements.txt

```diff
@@ -1,4 +1,3 @@
-ingenialink @ git+https://github.com/ingeniamc/ingenialink-python@develop
+ingenialink @ git+https://github.com/ingeniamc/ingenialink-python@INGK-802-support-custom-dictionaries-in-the-virtual-drive
```

Comment:
This change should be undone after merging https://github.com/ingeniamc/ingenialink-python/pull/361